gtk/src/auto/container.rs

```diff
@@ -203,6 +227,22 @@ impl<O: IsA<Container>> ContainerExt for O {
     //    unsafe { TODO: call ffi:gtk_container_child_set() }
     //}
 
+    fn child_set_property(
+        &self,
+        child: &impl IsA<Widget>,
+        property_name: &str,
+        value: &glib::Value,
+    ) {
+        unsafe {
+            ffi::gtk_container_child_set_property(
```

Comment:
This needs checking if the property exists, if the value is of the correct type, if the property is writable